proposals/texture-component-swizzle.md

```diff
@@ -55,6 +55,12 @@ The `GPUTexture.createView(descriptor)` algorithm is extended with the following
   - `descriptor.swizzle.b` must be `"b"`.
   - `descriptor.swizzle.a` must be `"a"`.
 
+## Validation
+
+A validation error happens if the swizzle is not the default and the `"texture-component-swizzle"` feature is enabled.
```

Comment:
I don't think that this is what the group previously agreed to. If we add new things to the IDL, passing incorrect values in it will result in errors if the feature is not enabled. This is the same as when a new IDL dictionary member is added and JS code that happened to use it previously might start getting a TypeError.

Reply:
I meant `the `"texture-component-swizzle"` feature is **not** enabled` sorry

Reply:
I'll wait for the group to discuss and approve https://github.com/gpuweb/gpuweb/issues/5264 before merging.